JitsiConferenceEventManager.js

```diff
@@ -29,6 +29,12 @@ function JitsiConferenceEventManager(conference) {
             conference.statistics.sendMuteEvent(track.isMuted(),
                 track.getType());
         });
+    conference.on(
```

Comment:
That is good question. I guess nowhere just like all other event listeners added by this class as there is no way to dispose this class gracefully. I can add FIXME, as it's too big effort to clean this up now.

Anyway it seems like it integral part of JitsiConference instance, so it will be invalid with it's parent. There is no way to reuse JitsiConference instance as far as I know.